pic-sure-api-war/src/main/java/edu/harvard/dbmi/avillach/service/PicsureBaseEntityService.java

```diff
@@ -102,6 +104,7 @@ public Response updateEntity(List<T> entities, BaseRepository baseRepository){
         for (T t : entities){
             boolean dbContacted = false;
             if (forAdd) {
+                t.setUuid(null);
                 baseRepository.persist(t);
```

Comment:
Just curious - why do you need to set the UUID to null?

Reply:
My thought is to provide some fault tolerance. Cuz users may not only uses this through a UI and they may copy paste user with accidentally put the uuid in it, in that case, I think maybe I should let them in as well cuz they have all other valid information